File: ListaDeLaCompra_copia/back_py/controller/logout_controller.py
from flask import session

def post_logout():
    try:
        if session.get('usuario_id'):
            session.clear()
            return {'mensaje': 'Sesion cerrada correctamente'}, 200
        else:
            return {'error': 'No hay sesion activa'}, 400
    except Exception:
        return {'error': 'Error interno del servidor'}, 500

File: ListaDeLaCompra_copia/back/src/controller/logout.controller.js
const postLogout = async (req, res) => {
  try {
    if (req.session && req.session.usuarioId) {
      req.session.destroy((err) => {
        if (err) {
          return res.status(500).json({ error: 'Error interno del servidor' });
        }
        res.status(200).json({ mensaje: 'Sesión cerrada correctamente' });
      });
    } else {
      res.status(400).json({ error: 'No hay sesión activa' });
    }
  } catch (error) {
    res.status(500).json({ error: 'Error interno del servidor' });
  }
};
module.exports = { postLogout };
